### DISCUSSION TITLE
jsPDF

### DISCUSSION THREAD

**Discussion by Mappelflap** [sidorares/node-mysql2#1641] [Q&A]
Hi there,

I am trying to figure out if it is possible to use jsPDF to create PDF's after a user pushes a button. 
Anyone got any suggestions on how to realize this? I have tried to use a HTML plugin (see below). 

Greetings and thanks in advance,

Mappelflap

[**HTML**
](https://stackoverflow.com/questions/16858954/how-to-properly-use-jspdf-library/43300193#43300193)

**Comment by sidorares** [comment]
I'm pretty sure its possible but this is completely wrong forum to discuss this topic, maybe try https://github.com/parallax/jsPDF/issues

**Reply by Mappelflap** [reply]
Hi Sidorares,

Thank you very much for your reply. I think I understand how to use jsPDF but I wonder if it is possible to execute this from Budibase. Are you sure it can be done?

Thanks again!

**Reply by sidorares** [reply]
I believe your question is related to jsPDF. Please use jsPDF issues/discussions or any other relevant forum for this question
